core/src/main/resources/plugin.yml

```diff
@@ -1,8 +1,9 @@
 name: LWC
 main: com.griefcraft.lwc.LWCPlugin
-version: ${project.version}
+version: 4.5.1-SNAPSHOT
```

Comment:
It appears to work fine when I compile it on my Jenkins: http://ci.dmulloy2.net/job/LWC/